src/main/webapp/js/instructorFeedbackEdit/customFeedbackPaths.js

```diff
@@ -0,0 +1,358 @@
+var FEEDBACK_PARTICIPANT_TYPE_CUSTOM = 'CUSTOM';
+var FEEDBACK_PARTICIPANT_TYPE_SELF = 'SELF';
+var FEEDBACK_PARTICIPANT_TYPE_STUDENTS = 'STUDENTS';
+var FEEDBACK_PARTICIPANT_TYPE_INSTRUCTORS = 'INSTRUCTORS';
+var FEEDBACK_PARTICIPANT_TYPE_TEAMS = 'TEAMS';
+var FEEDBACK_PARTICIPANT_TYPE_OWN_TEAM = 'OWN_TEAM';
+var FEEDBACK_PARTICIPANT_TYPE_OWN_TEAM_MEMBERS = 'OWN_TEAM_MEMBERS';
+var FEEDBACK_PARTICIPANT_TYPE_OWN_TEAM_MEMBERS_INCLUDING_SELF = 'OWN_TEAM_MEMBERS_INCLUDING_SELF';
+var FEEDBACK_PARTICIPANT_TYPE_NONE = 'NONE';
+var TEAM_NAME_INSTRUCTORS = 'Instructors';
+
+var sessionCreator;
+var studentEmailToTeamNameMap;
+var instructorEmails;
+var teamNameToStudentEmailsMap;
+var studentEmails;
+var teamNames;
+var allPossibleFeedbackGivers;
+var allPossibleFeedbackRecipients;
+
+$(document).ready(function() {
+    initialiseCustomFeedbackPathsData();
+    initialiseFeedbackPathsSpreadsheets();
+    bindEventHandlers();
```

Comment:
Since this is a whole new file, do you think we can namespace it. That way, we can have short method names without fear of populating global namespace.
E.g. this `bindEventHandlers()` may possibly confilct

More importantly, when adding methods, we'll consider whether it really belongs here. Also, when we eventually decide to revamp our js, I think a namespaced file is easier to convert